integr8ly/integreatly-operator
# test: update B03 test to include RHMI CR developer permission verification

# Issue link
https://issues.redhat.com/browse/MGDAPI-3449

# What
Updated B03 functional test to include RHMI CR developer permission verification. A developer user must not be able to access this or any other CR.

# Verification steps

- Run the B03 functional test: `INSTALLATION_TYPE=managed-api TEST=B03 BYPASS_STORAGE_TYPE_CHECK=true make test/e2e/single`
- Verify that 1 test passed and 53 were skipped
